frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,108 @@
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const NAMESPACE = 'openshift-storage';
+const KIND = 'storagecluster';
+const EXPANDWAIT = 150000;
+const storageCluster = JSON.parse(
+  execSync(`kubectl get -o json -n ${NAMESPACE} ${KIND}`).toString(),
+);
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+
+describe('Check add capacity functionality for ocs service', () => {
+  beforeAll(async () => {
+    // check whether the cluster is present or not, and check for its ready state
+    // if not present exit and console that no cluster is present, will be send in next PR
+
+    const { name } = storageCluster.items[0].metadata;
+    clusterJSON = JSON.parse(
+      execSync(`kubectl get -o json -n ${NAMESPACE} ${KIND} ${name}`).toString(),
+    );
+    previousCnt = _.get(clusterJSON, 'spec.storageDeviceSets[0].count', undefined);
+    const uid = _.get(clusterJSON, 'metadata.uid', undefined).toString();
+    previousPods = JSON.parse(execSync(`kubectl get pods -n ${NAMESPACE} -o json`).toString());
+
+    await browser.get(
+      `${appHost}/k8s/ns/openshift-storage/operators.coreos.com~v1alpha1~ClusterServiceVersion`,
+    );
+
+    await click(ocsOp);
+    await click(storageClusterView);
+
+    await browser.wait(until.presenceOf(storageClusterRow(uid)));
+    const kebabMenu = storageClusterRow(uid).$('button[data-test-id="kebab-button"]');
+
+    await click(kebabMenu);
+    await click(addCapacityLbl);
+    await click(addCapacityBtn);
+
+    updatedClusterJSON = JSON.parse(
+      execSync(`kubectl get -o json -n ${NAMESPACE} ${KIND} ${name}`).toString(),
+    );
+
+    updatedCnt = _.get(updatedClusterJSON, 'spec.storageDeviceSets[0].count', undefined);
+    const statusCol = storageClusterRow(uid).$('td:nth-child(4)');
+
+    // need to wait as cluster states fluctuates for some time. Waiting for 2 secs for the same
+    await browser.sleep(2000);
+
+    await browser.wait(until.textToBePresentInElement(statusCol, 'Progressing'));
+    await browser.wait(
+      until.textToBePresentInElement(statusCol.$('span.co-icon-and-text span'), 'Ready'),
+    );
+
+    updatedPods = JSON.parse(execSync(`kubectl get pod -o json -n ${NAMESPACE}`).toString());
+  }, EXPANDWAIT);
+
+  it('Newly added capacity should takes into effect at the storage level', () => {
+    // by default 2Tib capacity is being added
+    expect(updatedCnt - previousCnt).toEqual(1);
+  });
+
+  it('New osd pods corresponding to the additional capacity should be in running state', () => {
```

Comment:
Also the description says 
```
  it('New osd pods corresponding to the additional capacity should be in running state', () => {
```
but, I don't see anywhere that we are looking for RUNNING pods.